eng/testing/workloads-testing.targets

```diff
@@ -190,6 +190,7 @@
                      LocalNuGetsPath="$(LibrariesShippingPackagesDir)"
                      TemplateNuGetConfigPath="$(RepoRoot)NuGet.config"
                      SdkWithNoWorkloadInstalledPath="$(_SdkWithNoWorkloadPath)"
+                     SkipUpdateAppRefPack="true"
```

Comment:
This is temporal, right? And it will happen year again, right? If so, could we calculate whether to update app ref pack or not somehow?

Reply:
The runtime, and sdk repos are slightly differently matched in what gets changed till everything is good, so it's better to decide this when those changes happen, instead of having it done automatically.